crates/relayer-amplifier-api-integration/src/from_amplifier.rs

```diff
@@ -57,7 +52,7 @@ where
                 // in case we were awoken by join_set being ready, let's re-run this function,
                 // while returning the result of `internal`.
                 cx.waker().wake_by_ref();
-                return Poll::Ready(Some(Ok(res)));
+                return Poll::Ready(Some(Ok(Ok(()))));
```

Comment:
I’d prefer not to edit this file much, so I added Ok(Ok(())) here. Later, in the `JoinSet`, a future returning eyre::Result<()> will be added.